presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -355,3 +360,67 @@ Lambda Aggregate Functions
         -- (2, 42)
 
     The state type must be a boolean, integer, floating-point, or date/time/interval.
+
+
+Filtering During Aggregation
+----------------------------
+
+The ``FILTER`` keyword can be used to remove rows from aggregation processing
+with a condition expressed using a ``WHERE`` clause. This is evaluated for each
+row before it is used in the aggregation and is supported for all aggregate
+functions.
+
+    AGGREATE_FUNCTION(...) FILTER(WHERE <condition>)
+
+For example, you want to add a condition on the count for Iris flowers,
+modifying the following query.
+
+.. code-block:: none
+
+    SELECT species,
+           count(*) AS count
+    FROM memory.default.iris
+    GROUP BY species;
+
+.. code-block:: none
+
+    species    | count
+    -----------+-------
+    setosa     |   50
+    virginica      50
+    versicolor|    50
+
+If you just use a normal WHERE statement you loose information:
+
+.. code-block:: none
+
+    SELECT species,
+        count(*) AS count
+    FROM memory.default.iris
+    WHERE petallengthcm > 4
+    GROUP BY species;
+
+.. code-block:: none
+
+    species    | count
+    -----------+-------
+    virginica  |   50
+    versicolor     34
+
+Using a filter you retain all information:
```

Comment:
i had a comment, but i don't see it, maybe i didnt publish...

I'd just have one example showing filtered agg, instead of 3 examples (and only 1 showing that)

WDYT?

Reply:
This is only one example. There are three queries and results so readers can actually see what the difference in result is. I would leave it as is.